message_definitions/v1.0/common.xml

```diff
@@ -3406,9 +3400,6 @@
     </enum>
     <enum name="CAMERA_TRACKING_TARGET_DATA" bitmask="true">
       <description>Camera tracking target data (shows where tracked target is within image)</description>
-      <entry value="0" name="CAMERA_TRACKING_TARGET_DATA_NONE">
```

Comment:
This was removed upstream in https://github.com/mavlink/mavlink/pull/2212 - along with a number of other ones - worth a quick check to see the other ones in this don't need an update

Reply:
Done